app/src/main/res/layout/fragment_thread_list.xml

```diff
@@ -19,166 +19,203 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context=".ui.main.thread.ThreadListFragment">
-
-    <com.google.android.material.button.MaterialButton
-        android:id="@+id/menuDrawerButton"
-        style="@style/IconButton"
-        android:layout_width="32dp"
-        android:layout_height="40dp"
-        android:layout_marginStart="@dimen/marginStandardMedium"
-        android:layout_marginTop="@dimen/marginStandardMedium"
-        android:contentDescription="@string/contentDescriptionButtonMenu"
-        app:icon="@drawable/ic_burger"
-        app:layout_constraintBottom_toTopOf="@id/mailboxName"
-        app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:layout_constraintVertical_bias="0.0" />
+    android:layout_height="match_parent">
 
-    <include
-        android:id="@+id/searchViewCard"
-        layout="@layout/item_search_view"
+    <androidx.drawerlayout.widget.DrawerLayout
```

Comment:
use this layout as a root layout, it does not need a parent

Reply:
If we do that, we get an `IllegalArgumentException: DrawerLayout must be measured with MeasureSpec.EXACTLY`.

The solution was to put the `DrawerLayout` in a `ConstraintLayout`.

Reply:
What?? 🤔 this is not normal